### PR TITLE
Update docs for Zapier integrations

### PR DESCRIPTION
**Pull request by acoumb**
## Description

Updated docs for the following integrations:
- Umbraco.Cms.Integrations.Automation.Zapier `2.0.0`
- Umbraco.Forms.Integrations.Automation.Zapier `4.0.0`

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

- Umbraco.Cms.Integrations.Automation.Zapier `2.0.0`
- Umbraco.Forms.Integrations.Automation.Zapier `4.0.0`

## Deadline (if relevant)

Anytime

### CODE REVIEW

File: marketplace-and-integrations/integrations/zapier-with-umbraco-forms.md [hunk 1]
```diff
@@ -118,3 +118,16 @@ To enhance security, you can specify a User Group that the user connecting needs
...
 {% endtabs %}
+
+### Version 4.0.0 and up
+
+Version `4.0.0` of the integration adds some breaking changes to the Zaps creation flow with the new API endpoints and the Web Components-based Umbraco backoffice.
+
+As a result, users on Umbraco 14+ will need to use version `3.0.0` of the Umbraco Zapier application.
+
+{% hint style="info" %}
+Version `3.0.0` of the Umbraco app is available in the Zapier marketplace via the following invitation: https://zapier.com/developer/public-invite/157905/5f6dc86efe92c244cf0b2ff62af9d747/
```
Comment (sofietoft) [inline marketplace-and-integrations/integrations/zapier-with-umbraco-forms.md:129]:
```suggestion
Version `3.0.0` of the Umbraco app is available in the Zapier marketplace via the following invitation: [https://zapier.com/developer/public-invite/157905/5f6dc86efe92c244cf0b2ff62af9d747/](https://zapier.com/developer/public-invite/157905/5f6dc86efe92c244cf0b2ff62af9d747/)
```

File: marketplace-and-integrations/integrations/zapier.md [hunk 2]
```diff
@@ -118,3 +118,16 @@ The dashboard is composed of two sections:
...
 The _Trigger Webhook_ action will send a test request to the Zap trigger, enabling the preview of requests in the Zap setup workflow.
+
+### Version 2.0.0 and up
+
+Version `2.0.0` of the integration adds some breaking changes to the Zaps creation flow with the new API endpoints and the Web Components-based Umbraco backoffice.
+
+As a result, users on Umbraco 14+ will need to use version `3.0.0` of the Umbraco Zapier application.
+
+{% hint style="info" %}
+Version `3.0.0` of the Umbraco app is available in the Zapier marketplace via the following invitation: https://zapier.com/developer/public-invite/157905/5f6dc86efe92c244cf0b2ff62af9d747/
```
Comment (sofietoft) [inline marketplace-and-integrations/integrations/zapier.md:129]:
```suggestion
Version `3.0.0` of the Umbraco app is available in the Zapier marketplace via the following invitation: [https://zapier.com/developer/public-invite/157905/5f6dc86efe92c244cf0b2ff62af9d747/](https://zapier.com/developer/public-invite/157905/5f6dc86efe92c244cf0b2ff62af9d747/)
```

**Review by sofietoft — COMMENTED** [review]
I've added a couple of suggestions regarding the links.

I think it's a shame that this info is added to the bottom of the article.
If it's very important to use a specific version of Zapier for a specific Umbraco version, would it make sense to move this section to below the table with the "Minimum versions requirement" section further up in the articles?

**Comment by acoumb** [comment]
It makes sense, although I do not expect a high number of installs of Zapier on Umbraco 14. 
I've moved the section up below versioning.

Thanks @sofietoft

**Comment by sofietoft** [comment]
Perfect. Thanks! 🙌 

I'll get it merged.

**Pull request merged**